### ISSUE TITLE
how to update the words vectors

### ISSUE THREAD

**Issue by bruceli518** [dennybritz/cnn-text-classification-tf#87]
hi guys, according to the original paper made by Kim, the best way is to non-static, but I still do not how to implement it. 

I have tried some ways, like creating a graph in tensorflow and making the inputs as Variable, so that when you minimize the loss, the inputs can also be updated. 

But the memory of GPU requires you to load very small batch of data, you need to slice the inputs, unfortunately, the type of inputs will become 'tensor' after slice, it means that inputs will not be updated. 

I really want to know how to implement it.(Don't ask me to calculate the gradient by hand and then update it............)

**Comment by trideeprath** [comment]
By default it is non-static

**Non-Static**
`self.W = tf.Variable(tf.random_uniform([vocab_size, embedding_size], -1.0, 1.0), name="W")
`

**Static**
```
self.W = tf.Variable(tf.random_uniform([vocab_size, embedding_size], -1.0, 1.0), trainable=False, name="W")
```
